tests/models/markuplm/test_tokenization_markuplm.py

```diff
@@ -2195,12 +2194,11 @@ def test_only_label_first_subword(self):
 
         # TODO test fast tokenizer (see test in LayoutLMv2)
 
-    @slow
     def test_markuplm_integration_test(self):
 
         tokenizer_p = MarkupLMTokenizer.from_pretrained("microsoft/markuplm-base")
         # no fast tokenizer for now
-        # tokenizer_r = MarkupLMTokenizerFast.from_pretrained("microsoft/markuplm-base")
+        tokenizer_r = MarkupLMTokenizerFast.from_pretrained("microsoft/markuplm-base")
```

Comment:
@lockon-n , can you re-try with this change? :pray: 
```suggestion
        tokenizer_r = MarkupLMTokenizerFast.from_pretrained("microsoft/markuplm-base", from_slow=True)
```
If it solves the issue it's just because the `ŧokenizer.json` file at https://huggingface.co/microsoft/markuplm-base needs to be updated

Reply:
Still fail:(

Reply:
Oh :disappointed: 

Could this be coming from the `xpath_tags_seq` list?

I have trouble remembering how the nodes and xpaths need to be tied. I particular I don't remember why line 102 is `xpaths = ["/html/body/div/li[1]/div/span", "/html/body/div/li[1]/div/span"] ` and not `xpaths = ["/html/body/div/li[1]/div/span"] `

https://github.com/NielsRogge/transformers/blob/485b2ca08185231e88d228797e1103ebb43fb23b/tests/models/markuplm/test_tokenization_markuplm.py#L99-L112